src/package/packageVersionList.ts

```diff
@@ -14,19 +14,40 @@ import { PackageVersionListResult, PackageVersionListOptions, ListPackageVersion
 Messages.importMessagesDirectory(__dirname);
 const messages = Messages.loadMessages('@salesforce/packaging', 'package_version_create');
 
-// Stripping CodeCoverage, HasPassedCodeCoverageCheck as they are causing a perf issue in 47.0+ W-6997762
-const DEFAULT_SELECT =
-  'SELECT Id, Package2Id, SubscriberPackageVersionId, Name, Package2.Name, Package2.NamespacePrefix, Package2.IsOrgDependent, ' +
-  'Description, Tag, Branch, MajorVersion, MinorVersion, PatchVersion, BuildNumber, IsReleased, ' +
-  'CreatedDate, LastModifiedDate, IsPasswordProtected, AncestorId, ValidationSkipped, CreatedById ' +
-  'FROM Package2Version';
-
-const VERBOSE_SELECT =
-  'SELECT Id, Package2Id, SubscriberPackageVersionId, Name, Package2.Name, Package2.NamespacePrefix, ' +
-  'Description, Tag, Branch, MajorVersion, MinorVersion, PatchVersion, BuildNumber, IsReleased, ' +
-  'CreatedDate, LastModifiedDate, IsPasswordProtected, CodeCoverage, HasPassedCodeCoverageCheck, AncestorId, ValidationSkipped, ' +
-  'ConvertedFromVersionId, Package2.IsOrgDependent, ReleaseVersion, BuildDurationInSeconds, HasMetadataRemoved, CreatedById ' +
-  'FROM Package2Version';
+const defaultFields = [
```

Comment:
@sethmuss maybe add a UT or NUTs changes for query construction

Reply:
@peternhale this is the same review from yesterday you signed off on. Only redo-ing due to commit signing issues. We've got some WI's for adding NUTS that we'll be working on soon.